packages/core/core/src/public/Asset.js

```diff
@@ -193,6 +194,10 @@ class BaseAsset {
   getMapBuffer(): Promise<?Buffer> {
     return this.#asset.getMapBuffer();
   }
+
+  getSourcesContent(): Promise<?SourcesContentDictionary> {
```

Comment:
Should this be getSourceContent rather than plural? Or even just getSource? We'd have getCode which gets the compiled code and getSource which gets the source code?

Reply:
but it can return multiple sources? in case of a bundled library?

Reply:
Ooh didn't realize that